src/assets/formats/textures.rs

```diff
@@ -12,7 +12,7 @@ pub use imagefmt::Error as ImageError;
 /// ImageData provided by formats, can be interpreted as a texture.
 #[derive(Clone, Debug)]
 pub struct ImageData {
-    raw: Image<u8>,
+    pub(crate) raw: Image<u8>,
```

Comment:
Because then we'd be exposing a type from imagefmt, which I'm not sure if we want to lock that crate in as part of our public API.